cluster/config-defaults.yaml

```diff
@@ -121,6 +121,7 @@ zmon_agent_replicas: "0"
 zmon_aws_agent_replicas: "0"
 zmon_redis_replicas: "0"
 zmon_scheduler_replicas: "0"
+zmon_scheduler_mem: "0Gi"
```

Comment:
is this correct, zero gigs

Reply:
This is for e2e tests. It doesn't really matter since replicas is set to `0` anyway